qa/smoke-test-plugins/build.gradle

```diff
@@ -33,6 +33,10 @@ testClusters.integTest {
       // Do not attempt to install ingest-attachment in FIPS 140 as it is not supported (it depends on non-FIPS BouncyCastle)
       return
     }
+    if (pluginName == 'quota-aware-fs') {
+      // This plugin has to be configured to work via system properties
```

Comment:
We should just be skipping all bootstrap plugins rather than hardcoding `quota-aware-fs`?